### PR TITLE
[WC-1025] Update carousel documentation

### PR DESCRIPTION
**Pull request by raffiar**
@ConnorLand Mentioning you because of our previous discussion.
This pr needs technical review

### CODE REVIEW

File: content/appstore/widgets/carousel.md [hunk 1]
```diff
@@ -12,6 +12,57 @@ The [Carousel](https://marketplace.mendix.com/link/component/47784/) widget disp
...
+* Supports widget as a content
+* Navigate to the next or previous image using arrows or dots
+* Autoplay your images
+* Set the timeout between each slide
+* Use the infinite loop to scroll the slides endlessly
+* Add animation when switching slides
+
+## 2 Usage
+
+The widget requires a context configured on the **Data Source** section in **General** tab. The possible data sources are described below.
```
Comment (diego-antonelli) [inline content/appstore/widgets/carousel.md:24]: Sounds a bit confusing the context of a data source.. Maybe we should change to `The widget requires a defined **Data source** available in the **General** tab.`

Comment (diego-antonelli) [reply content/appstore/widgets/carousel.md:24]: But I guess that is for docs team. Technical check is OK

Comment (ConnorLand) [reply content/appstore/widgets/carousel.md]: I like the way you phrased it in the comment @diego-antonelli . Change is done.

File: content/appstore/widgets/carousel.md [hunk 2]
```diff
@@ -12,6 +12,57 @@ The [Carousel](https://marketplace.mendix.com/link/component/47784/) widget disp
 
 ### 1.1 Features
 
+* Allows you to use the widget as a content area
```
Comment (ConnorLand) [inline content/appstore/widgets/carousel.md:15]: Is "content area" correct @raffiar ? I didn't know what "as a content" meant.

Comment (raffiar) [reply content/appstore/widgets/carousel.md:15]: hmm, that's a good question. What I meant is that you can put any widget in carousel. Like the slide can be not only image (widget) but also text (widget)
What is the best way to explain that?

Comment (diego-antonelli) [reply content/appstore/widgets/carousel.md:15]: I think we use the term "Composable region"

https://docs.mendix.com/appstore/widgets/tooltip#1-1-trigger
https://docs.mendix.com/appstore/modules/tree-node#general

Comment (raffiar) [reply content/appstore/widgets/carousel.md:15]: maybe it's better to say `Allows you to use a widget as an item content`?

Comment (ConnorLand) [reply content/appstore/widgets/carousel.md]: I've resolved it by using both language suggestions from Raffi and Diego.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**